src/otx/core/exporter/base.py

```diff
@@ -68,6 +68,7 @@ def export(
         base_model_name: str = "exported_model",
         export_format: OTXExportFormatType = OTXExportFormatType.OPENVINO,
         precision: OTXPrecisionType = OTXPrecisionType.FP32,
+        export_args: dict[str, Any] | None = None,
```

Comment:
Personally, this should be cautious to introduce new arguments. If it is limited to the specific arguments, e.g., `input`, `input_model`, or `example_input`, it could be better to introduce them directly instead of using dict type `export_args`. Is there any example or candiate of `export_args` during `openvino.convert_model`?

Reply:
`self.to_onnx` was only able to support naive dummy image input for simple tasks, which was not for other complicate tasks like visual prompting, and static arguments such as `verbose` and `opset`, not inputs.
`export_args` is to support dynamic arguments like inputs in `torch.onnx.export` in `self.to_onnx`.
And it is expanded to other exports to fit argument information.
For `openvino.convert_model`, we can use this argument like below
https://github.com/openvinotoolkit/training_extensions/blob/2708d7ba16465e35cd7665da3043223ecc2dde3f/src/otx/core/exporter/native.py#L81-L87

Reply:
I think we need to extend export_args description emphasizing that we can modify sample input. Also, if it is only needed for example_input, then I propose to create `example_input` parameter instead of `export_args`. `export_args` has unclear naming and unclear purpose. "input" always can be obtained from self.input_size, no need to add it in arguments.

Reply:
I'm not sure about your comment, but I just patched previous implementations.
https://github.com/openvinotoolkit/training_extensions/blob/c8c3cae11d0702e707fd17132aca2de90e61a0a2/src/otx/core/exporter/native.py#L72-L76
And if so, I'll remove `input` in `export_args`. Is it fine?

Reply:
> I think we need to extend export_args description emphasizing that we can modify sample input. Also, if it is only needed for example_input, then I propose to create `example_input` parameter instead of `export_args`. `export_args` has unclear naming and unclear purpose. "input" always can be obtained from self.input_size, no need to add it in arguments.

I updated the name from `export_args` to `example_inputs` at https://github.com/openvinotoolkit/training_extensions/pull/2955/commits/27a05cd95b0acc2e3f7db37c5a1bbaf1adc524a5.